risc0/build/src/lib.rs

```diff
@@ -382,7 +382,18 @@ fn build_guest_package<P>(
 
     let mut cmd = Command::new(cargo);
     let mut child = cmd
-        .env("CARGO_ENCODED_RUSTFLAGS", "-C\x1fpasses=loweratomic")
+        .env(
+            "CARGO_ENCODED_RUSTFLAGS",
+            [
+                // Replace atomic ops with nonatomic versions since the guest is single threaded.
+                "-C",
+                "passes=loweratomic",
```

Comment:
Quick question: would this impact our crate compatibility at all? Or will it just effect the deeper lowering phase?

Reply:
It just affects the lowering phase, and the "passes=loweratomic" was already there; I just reformatted some. :)